zipline/utils/sentinel.py

```diff
@@ -27,9 +33,8 @@ def sentinel(name, doc=None):
         ) % (name, value.__doc__, doc))
 
     @object.__new__   # bind a single instance to the name 'Sentinel'
-    class Sentinel(object):
+    class Sentinel(_Sentinel):
```

Comment:
I moved `is_sentinel` up to the top of this file to make it clearer that the base class exists so that we can use `isinstance` for `is_sentinel`.